extensions/undertow-websockets/deployment/src/main/java/io/quarkus/undertow/websockets/deployment/HotReplacementWebsocketEndpoint.java

```diff
@@ -36,6 +36,7 @@
     static final String QUARKUS_HOT_RELOAD = "/quarkus/live-reload";
     static final String QUARKUS_SECURITY_KEY = "quarkus-security-key";
     static final String QUARKUS_HOT_RELOAD_PASSWORD = "quarkus.live-reload.password";
+    static final String QUARKUS_HOT_RELOAD_PASSWORD_ENV = "QUARKUS_LIVE_RELOAD_PASSWORD";
```

Comment:
Should we rename the `_HOT_RELOAD` constants to `_LIVE_RELOAD` (it can be done in a separate commit/pr)

Reply:
Good idea, we use that terminology elsewhere so it makes sense to be consistent

Reply:
Done